public/app/features/dashboard-scene/scene/PanelMenuBehavior.tsx

```diff
@@ -54,7 +55,7 @@ export function panelMenuBehavior(menu: VizPanelMenu) {
         text: t('panel.header-menu.view', `View`),
         iconClassName: 'eye',
         shortcut: 'v',
-        onClick: () => reportInteraction('dashboards_panelheader_menu', { item: 'view' }),
+        onClick: () => DashboardInteractions.panelMenuItemClicked({ item: 'view' }),
```

Comment:
I'd recommend forcing the item to be passed through the function to ensure it is properly used. Also, I'd add an enum with the items and move them to the analytics file, so we know what items are tracked from a single place.
```ts
DashboardInteractions.panelMenuItemClicked(item: MenuItems, metadata: object)
```